rules/CodeQuality/Rector/If_/ConsecutiveNullCompareReturnsToNullCoalesceQueueRector.php

```diff
@@ -142,7 +142,7 @@ public function refactor(Node $node): ?Node
                 }
 
                 $node->stmts = $originalStmts;
-                return $node;
+                return null;
```

Comment:
If we roll back to the existing original statements we return null to indicate that no change has been made